src/proc/rotation-filter.h

```diff
@@ -0,0 +1,41 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include "../include/librealsense2/hpp/rs_frame.hpp"
+#include "../include/librealsense2/hpp/rs_processing.hpp"
+#include "proc/synthetic-stream.h"
+
+namespace librealsense
+{
+
+    class rotation_filter : public stream_filter_processing_block
+    {
+    public:
+        rotation_filter();
+
+    protected:
+        rs2::frame prepare_target_frame(const rs2::frame& f, const rs2::frame_source& source, rs2_extension tgt_type);
+
+        template< size_t SIZE >
+        void rotate_depth( uint8_t * const out, const uint8_t * source, int width, int height );
+
+        rs2::frame process_frame(const rs2::frame_source& source, const rs2::frame& f) override;
+
+    private:
+        void    update_output_profile(const rs2::frame& f);
+
+        
+        int                _control_val;
```

Comment:
It is misaligned with the next like.
Normally you choose not to align and keep 1 space or align to next lines.
Here it's a mixure?

Reply:
Fixed